alicloud/resource_alicloud_emr_cluster_test.go

```diff
@@ -0,0 +1,161 @@
+	return nil
+}
+
+func TestAccAlicloudEmrCluster_basic(t *testing.T) {
+	var v *emr.DescribeClusterV2Response
+	resourceId := "alicloud_emr_cluster.default"
+	ra := resourceAttrInit(resourceId, nil)
+	rc := resourceCheckInit(resourceId, &v, func() interface{} {
+		return &EmrService{testAccProvider.Meta().(*connectivity.AliyunClient)}
+	})
+	rac := resourceAttrCheckInit(rc, ra)
+
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	rand := acctest.RandIntRange(1000, 9999)
+	name := fmt.Sprintf("tf-testAccEmrClusterConfig%d", rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourceEmrClusterConfigDependence)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		//CheckDestroy:  testAccAlicloudEmrClusterDestroy,
+		CheckDestroy: rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				//Config: testAccEMRClusterConfig_basic,
+				Config: testAccConfig(map[string]interface{}{
+					"name":                      name,
+					"emr_ver":                   "EMR-3.22.0",
+					"cluster_type":              "HADOOP",
+					"deposit_type":              "HALF_MANAGED",
+					"high_availability_enable":  "true",
+					"option_software_list":      []string{"HBASE", "PRESTO"},
+					"zone_id":                   "cn-huhehaote-a",
+					"security_group_id":         "sg-hp39gm8n17f3u0iiv4e7",
+					"is_open_public_ip":         "true",
+					"charge_type":               "PostPaid",
+					"vpc_id":                    "vpc-hp32dbx98fhcjp9kr5m75",
+					"vswitch_id":                "vsw-hp34pmwcw8i5was4sn07h",
+					"net_type":                  "vpc",
+					"user_defined_emr_ecs_role": "EMRUserDefineRole-Role1",
+					"ssh_enable":                "true",
+					"master_pwd":                "ABCtest1234!",
+					"io_optimized":              "true",
+
+					"host_group": []map[string]interface{}{
+						map[string]interface{}{
+							"host_group_type":   "MASTER",
+							"node_count":        "2",
+							"instance_type":     "ecs.g5.xlarge",
+							"disk_type":         "CLOUD_SSD",
+							"disk_capacity":     "80",
+							"disk_count":        "1",
+							"sys_disk_type":     "CLOUD_SSD",
+							"sys_disk_capacity": "80",
+						},
+						map[string]interface{}{
+							"host_group_type":   "CORE",
+							"node_count":        "3",
+							"instance_type":     "ecs.g5.xlarge",
+							"disk_type":         "CLOUD_SSD",
+							"disk_capacity":     "80",
+							"disk_count":        "4",
+							"sys_disk_type":     "CLOUD_SSD",
+							"sys_disk_capacity": "80",
+						},
+					},
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"name":         name,
+						"emr_ver":      "EMR-3.22.0",
+						"cluster_type": "HADOOP",
+						"zone_id":      "cn-huhehaote-a",
+						"charge_type":  "PostPaid",
+						"net_type":     "vpc",
+					}),
+				),
+			},
```

Comment:
Please add importer test and changing case

Reply:
> Please add importer test and changing case

not supported now.

Reply:
But the resource has supported the `Importer`